cli/src/funTest/kotlin/ExamplesFunTest.kt

```diff
@@ -162,6 +170,35 @@ class ExamplesFunTest : StringSpec() {
             report shouldHaveSize 1
         }
 
+        "emails can be send using notifications.kts" {
```

Comment:
I would change this to match the other test names:
`Notifications.kts can be complied and executed`